tests/templates/DonationForm.html.twig

```diff
@@ -0,0 +1,18 @@
+Amount: {$ betrag $}
+Payment type: {$ zahlweise $}
+Address type: {$ adresstyp $}
+Interval: {$ periode $}
+IBAN: {$ iban $}
+BIC: {$ bic $}
+Bank name: {$ bankName $}
+Address type: {$ adressType $}
+Salutation: {$ anrede $}
+Title: {$ titel $}
+Company: {$ firma $}
+First name: {$ vorname $}
+Last name: {$ nachname $}
+Street: {$ strasse $}
+Postal code: {$ plz $}
+City: {$ ort $}
+Country code: {$ country $}
+Email address: {$ email $}
```

Comment:
This is for the system test to make sure all required data is passed to the presenter. The actual form will use the parameters to populate/restore the form fields with the given values so users don't have to fill the form again.